systemtest/src/main/java/io/strimzi/systemtest/utils/kubeUtils/controllers/StatefulSetUtils.java

```diff
@@ -193,4 +193,22 @@ public static void waitForKafkaStatefulSetLabelsDeletion(String statefulSet, Str
             LOGGER.info("Kafka statefulSet label {} change to {}", labelKey, null);
         }
     }
+
+    public static void waitForNoRollingUpdate(String statefulSetName, Map<String, String> pods) {
+        // alternative to sync hassling AtomicInteger one could use an integer array instead
+        // not need to be final because reference to the array does not get another array assigned
+        int[] i = {0};
+
+        TestUtils.waitFor("Waiting for stability of rolling update will be not triggered", Constants.GLOBAL_POLL_INTERVAL, Constants.GLOBAL_TIMEOUT,
+            () -> {
+                if (!StatefulSetUtils.ssHasRolled(statefulSetName, pods)) {
+                    LOGGER.info("{} pods not rolling waiting, remaining seconds for stability {}", pods.toString(),
```

Comment:
Maybe we should use better log text like :`{} pods didn't roll. Remaining seconds for stability: {}`  wdyt?